### PR TITLE
[DCJ-188] Refactor Translator.js to JSX

### PR DESCRIPTION
**Pull request by PintoGideon**
### Addresses

https://broadworkbench.atlassian.net/browse/DCJ-188

### Summary

1. Remove the use of react-hyperscript-helpers
2. Update the file extension from js to jsx

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/Translator.jsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+              <div className='col-lg-12' style={{ backgroundColor: 'white' }}>
+                <h2>Translation Results</h2>
+              </div>
+            </div>
+            {isLoading && (
+              <div className='flex-row' style={{ width: '100%', justifyContent: 'center' }}>
+                <Spinner />
+              </div>
+            )}
+            {(error!==undefined && error!==null) && (
```
Comment (fboulnois) [inline src/pages/Translator.jsx]:
```suggestion
            {(error !== undefined && error !== null) && (
```

nit: spacing for clarity

**Review by fboulnois — APPROVED** [review]
Looks good, one suggestion:

**Review by rushtong — APPROVED** [review]
Tested locally 👍🏽  I agree with Florian's feedback as well.

**Review by aarohinadkarni — APPROVED** [review]
tested locally looks good 👍

**Pull request merged**